app/views/posts/_editor.haml

```diff
@@ -1,3 +1,4 @@
+- character_galleries = item.character.galleries.includes(:ordered_icons).ordered if item.character && current_user.icon_picker_grouping? && item.character.galleries.count > 1
```

Comment:
Provides `character_galleries` for if it's ever used, including `ordered_icons` since that's used in any code path that uses  `character_galleries`, it seems.